### ISSUE TITLE
No module named 'utils'  when trying to run 'gen_hha.py'

### ISSUE THREAD

**Issue by SnorlaxSE** [jackyjsy/CVPR21Chal-SLR#6]

**Comment by SnorlaxSE** [comment]
```
# from utils.rgbd_util import *
# from utils.getCameraParam import *
# from getHHA import getHHA
```
Just  remove these `from pkg `  line.

**Comment by jackyjsy** [comment]
Thanks for pointing out! It's been corrected in commit 24509a702318001f1148fef78f74d05df37cf8a1
